src/DateRangePicker.jsx

```diff
@@ -276,7 +275,7 @@ const DateRangePicker = createClass({
     if (selectionType === 'range') {
       if (selectedStartDate) {
         this.completeRangeSelection();
-      } else if (!this.isDateDisabled(date) && this.isDateSelectable(date)) {
+      } else if (date && !this.isDateDisabled(date) && this.isDateSelectable(date)) {
```

Comment:
Is this fixing a bug, or is there an implementation change here? :)

Reply:
Hmmm - most likely it was a bug but in hindsight it would be better to guard the call itself? I don't remember, I'll take it out.

Reply:
@AlanFoster Hmmm taking it out results in https://travis-ci.org/onefinestay/react-daterange-picker/builds/331594676#L867 - not sure what the best approach to fix that one is. This guard seems a bit arbitrary…